app/components/post_list/post_list.js

```diff
@@ -201,8 +206,12 @@ export default class PostList extends PureComponent {
     };
 
     handleScrollToIndexFailed = () => {
-        this.hasDoneInitialScroll = false;
-        this.scrollToInitialIndexIfNeeded(1, 1);
+        requestAnimationFrame(() => {
```

Comment:
Can we kill this on when component is unmounting? 

Assign this to a variable and call `cancelAnimationFrame` when it is going to unmount?

Reply:
@sudheerDev done